src/helpers/metrics-playground-opener.js

```diff
@@ -1,8 +1,8 @@
-import { getEnvironmentFromUrl } from './get-environment-from-url'
+import { getEnvironment } from '@/helpers'
 
 export const metricsPlaygroundOpener = () => {
-  const environment = getEnvironmentFromUrl(window.location.href)
-  const subdomain = environment === 'prod' ? 'manager' : 'stage-manager'
+  const environment = getEnvironment()
+  const subdomain = environment === 'production' ? 'manager' : 'stage-manager'
```

Comment:
Acho que podemos ter um método que retorne IsProduction algo do tipo.
```suggestion
  const subdomain = isProduction() ? 'manager' : 'stage-manager'
```